targets/bmv2/fuzzer.cpp

```diff
@@ -9,7 +11,102 @@ const Bmv2V1ModelProgramInfo &Bmv2V1ModelFuzzer::getProgramInfo() const {
     return *P4RuntimeFuzzer::getProgramInfo().checkedTo<Bmv2V1ModelProgramInfo>();
 }
 
-InitialP4RuntimeConfig Bmv2V1ModelFuzzer::produceInitialConfig() { return {}; }
+void addTernary(p4::v1::TableEntry *protoEntry, int fieldId, int bitwidth) {
+    auto value = Utils::getRandConstantForWidth(bitwidth)->value;
+    std::optional<std::string> valueStr = P4::ControlPlaneAPI::stringReprConstant(value, bitwidth);
```

Comment:
This needs to be checked. The optional may indicate an error.

Reply:
Yes, ideally propagate it upwards. You can use `std::optional` for it because we do not yet support `std::expected`.